queuebacca-core/src/main/java/io/axonif/queuebacca/Logger.java

```diff
@@ -55,6 +57,15 @@ public void error(String format, Object... arguments) {
 		log(format, arguments, logger::error);
 	}
 
+	public void initalizeContext(Context context) {
```

Comment:
`setContext()` I think makes more sense. Initialize seems to be more like an upfront, one time thing.